Reporting: Add theme option (#95008)
grafana/grafana: docs/sources/dashboards/create-reports/index.md

You can customize the branding options.

> ### Attachment settings:

Reviewer: ```suggestion
### Attachment settings
```

#### PDF

- **Company logo:** Company logo displayed in the report PDF. It can be configured by specifying a URL, or by uploading a file. The maximum file size is 16 MB. Defaults to the Grafana logo.

- **Theme:** Theme of the PDF attached to the report. Defaults to `Light`.

#### Embedded Image

- **Theme:** Theme of the dashboard image embedded in the email. Defaults to `Dark`.

> ### Email branding:

Reviewer: ```suggestion
### Email branding
```

> - **Theme:** Theme of the PDF attached to the report. Defaults to `Light`. Selected theme will also be applied to Preview PDF and Export as PDF. In the case when `Current` is selected, the PDF in the report will be in the admin's instance theme while Preview PDF and Export as PDF will be in the user's instance theme.

Reviewer: ```suggestion
- **Theme:** Theme of the PDF attached to the report. Defaults to **Light**. The selected theme is also applied to the PDFs generated when you click **Preview PDF** and **Export as PDF** during report creation. If **Current** is selected, the PDF in the report will be in the Admin's instance theme, but the preview and exported PDFs will be in the user's instance theme.
```

> - **Theme:** Theme of the dashboard image embedded in the email. Defaults to `Dark`.

Reviewer: ```suggestion
- **Theme:** Theme of the dashboard image embedded in the email. Defaults to **Dark**.
```

You can customize the branding options.

### Attachment settings

#### PDF

> - **Company logo:** Company logo displayed in the report PDF. It can be configured by specifying a URL, or by uploading a file. The maximum file size is 16 MB. Defaults to the Grafana logo.

Reviewer: ```suggestion
- **Company logo URL:** Company logo displayed in the report PDF. It can be configured by specifying a URL, or by uploading a file. The maximum file size is 16 MB. Defaults to the Grafana logo.
```

- **Theme:** Theme of the PDF attached to the report. Defaults to **Light**. The selected theme is also applied to the PDFs generated when you click **Preview PDF** during report creation or select the **Export as PDF** option on a dashboard. If **Current** is selected, the PDF in the report will be in the Admin's instance theme, but the preview and exported PDFs will be in the user's instance theme.

#### Embedded Image

- **Theme:** Theme of the dashboard image embedded in the email. Defaults to **Dark**.

### Email branding

> - **Company logo:** Company logo displayed in the report email. It can be configured by specifying a URL, or by uploading a file. The maximum file size is 16 MB. Defaults to the Grafana logo.

Reviewer: ```suggestion
- **Company logo URL:** Company logo displayed in the report email. It can be configured by specifying a URL, or by uploading a file. The maximum file size is 16 MB. Defaults to the Grafana logo.
```
